glcode.js

```diff
@@ -1,18 +1,33 @@
 Qt.include("three.js")
+Qt.include("controls.js")
+Qt.include("math.js")
+
 var camera, scene, renderer;
 var cube;
 var mesh;
+var controls;
 
 function initializeGL(canvas) {
-    camera = new THREE.PerspectiveCamera(27, canvas.innerWidth/canvas.innerHeight, 1, 4000);
-    camera.position.z = 2800;
+    camera = new THREE.PerspectiveCamera(27, 640/480, 1, 11000);
```

Comment:
my bad, I was debugging because the aspectRatio was not being set properly.
Fixed.